crates/transaction-fuzzer/tests/p2p_fuzz.proptest-regressions

```diff
@@ -6,3 +6,4 @@
x(Index(2683638568002300817))], reverse: true }, amount: 59031829, gas: 0, gas_price: 4691 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(12002830663853047437)), Index(Index(11643593263912374696))], reverse: false }, amount: 45872060, gas: 1999, gas_price: 3572 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(11063051103812160956)), Index(Index(2975621676637172629))], reverse: false }, amount: 6382882, gas: 49999999999, gas_price: 18446744071733551 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(10188218973691825014)), Index(Index(3586372881300142967))], reverse: true }, amount: 60915996, gas: 2001, gas_price: 100000 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(3404792176268814150)), Index(Index(5574009025926990150))], reverse: true }, amount: 66566081, gas: 50000000001, gas_price: 0 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(16793528402671529428)), Index(Index(11206880130786982694))], reverse: false }, amount: 60013796, gas: 2001, gas_price: 1496 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(11556772702320087302)), Index(Index(8741768899400628990))], reverse: false }, amount: 15081659, gas: 18446744073709551615, gas_price: 100000 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(13265850172203225897)), Index(Index(6503986099330455724))], reverse: true }, amount: 79762264, gas: 1999, gas_price: 18446744071733551 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(12318957970289835538)), Index(Index(432578061685274146))], reverse: true }, amount: 92756996, gas: 1050485, gas_price: 100001 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(4440034413518242351)), Index(Index(8721195220657765398))], reverse: true }, amount: 48715523, gas: 0, gas_price: 9676 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(5052583361844423489)), Index(Index(2592793421662325328))], reverse: false }, amount: 14764112, gas: 1999, gas_price: 18446744071733551 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(12801651744872746818)), Index(Index(16079878851640345577))], reverse: false }, amount: 64448742, gas: 2000, gas_price: 18446744071733550 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(1330288871122216839)), Index(Index(1693731629996493705))], reverse: true }, amount: 11663817, gas: 2090321, gas_price: 100000 }, P2PTransferGenRandomGasRandomPrice { sender_receiver: AccountPairGen { pair: [Index(Index(3265470854798872278)), Index(Index(10378431903077794562))], reverse: false }, amount: 41598738, gas: 18446744073709551614, gas_price: 100001 }]
```

Comment:
Is this a new test failure we've found?

Reply:
ah! This was one that I found in the old one and fixed, but forgot to commit the updated proptests in that commit